clients/drcachesim/common/options.cpp

```diff
@@ -561,6 +562,18 @@ droption_t<bytesize_t>
                  "of being simulated.  This skipping may be slow for large skip values; "
                  "consider -skip_instrs for a faster method of skipping.");
 
+droption_t<uint64_t> op_skip_to_timestamp(
+    DROPTION_SCOPE_FRONTEND, "skip_to_timestamp", 0, "Timestamp to start at",
+    "Specifies a timestamp to start at, skipping over prior records in the trace. "
+    "This is cross-cutting across all threads.  If the target timestamp is not "
+    "present as a timestamp marker, interpolation is used to approximate the "
+    "target location in each thread.  If -skip_instrs is also specified, this option "
```

Comment:
It would be nicer to crash if both -skip_isntrs and -skip_to_timestamp are specified. That's harder to miss than this comment.

Reply:
Done.